src/api/app/models/issue_tracker/issue_tracker_helper.rb

```diff
@@ -0,0 +1,33 @@
+class IssueTracker::IssueTrackerHelper
+  attr_reader :tracker, :issue_id
+  attr_accessor :url, :summary
+
+  def initialize(issue_id)
+    if issue_id.start_with?('CVE-')
+      @tracker = 'cve'
+      @issue_id = issue_id
+    else
+      @tracker, @issue_id = issue_id.split('#')
+    end
+  end
+
+  def bug
+    @issue_id.starts_with?('CVE-') ? @issue_id : @tracker + '#' + @issue_id
+  end
+
+  def valid?
+    !@issue_id.nil?
+  end
+
+  def cve?
+    @tracker == 'cve'
+  end
+
+  def not_cve?
+    !cve?
+  end
+
+  def to_s
```

Comment:
IMO this should have a different naming :thinking: , because is return an array an not a string, maybe `details` or `track_details`